common/fileview2/fverror.hpp

```diff
@@ -18,41 +18,43 @@
 #ifndef FVERROR_HPP
 #define FVERROR_HPP
 
-#include "jerrorrange.hpp"
+#include "errorlist.h"
 
-#define ERR_FILEVIEW_FIRST  2000
-#define ERR_FILEVIEW_LAST   2049
-
-#define FVERR_CouldNotResolveX                  2000
```

Comment:
I don't like this change. If I need to look up what error code 2002 means, I won't find it